tools/c7n_azure/c7n_azure/resources/key_vault_keys.py

```diff
@@ -158,3 +158,37 @@ def _process_resource_set(self, resources, event):
                 log.warning(error)
 
         return matched
+
+@KeyVaultKeys.filter_registry.register('keyvault-keys-rotation')
```

Comment:
suggestion, we don't need to prefix filters with the name of the resource, they are already namespaced, let's just call it `rotation` or `rotation-policy`